npm/extensions.bzl

```diff
@@ -297,21 +297,40 @@ def _npm_import_attrs():
 
     return attrs
 
-def _npm_exclude_package_contents_attrs():
-    return {
-        "package": attr.string(
-            doc = "Package name to apply exclusions to. Supports wildcards like '*' for all packages.",
-            mandatory = True,
-        ),
-        "patterns": attr.string_list(
-            doc = "List of glob patterns to exclude from the specified package.",
-            default = [],
-        ),
-        "use_defaults": attr.bool(
-            doc = "Whether to use default exclusion patterns for the specified package. Defaults are as to Yarn autoclean: https://github.com/yarnpkg/yarn/blob/7cafa512a777048ce0b666080a24e80aae3d66a9/src/cli/commands/autoclean.js#L16",
-            default = False,
-        ),
-    }
+_EXCLUDE_PACKAGE_CONTENT_ATTRS = {
+    "package": attr.string(
+        doc = "Package name to apply exclusions to. Supports wildcards like '*' for all packages.",
+        mandatory = True,
+    ),
+    "patterns": attr.string_list(
+        doc = "List of glob patterns to exclude from the specified package.",
+        default = [],
+    ),
+    "use_defaults": attr.bool(
+        doc = "Whether to use default exclusion patterns for the specified package. Defaults are as to Yarn autoclean: https://github.com/yarnpkg/yarn/blob/7cafa512a777048ce0b666080a24e80aae3d66a9/src/cli/commands/autoclean.js#L16",
+        default = False,
+    ),
+}
+
+_EXCLUDE_PACKAGE_CONTENT_DOCS = """Configuration for excluding package contents from npm packages.
```

Comment:
This one is new, a modification of what was previously documented on `npm_translate_lock(exclude_package_contents)`

Reply:
Should we switch the auto clean default to True above for 3.0?

Reply:
Not in this PR, but that is on [the list](https://github.com/aspect-build/rules_js/issues/1993)